modules/mod_authentication/templates/_logon_login_admin_form_fields.tpl

```diff
@@ -1,24 +1,29 @@
-<div class="form-group">
-    <label for="username" class="control-label">{_ Username _}</label>
-    <input class="form-control" type="text" id="username" name="username" value="" autofocus="autofocus" autocomplete="on" />
-    {% validate id="username"
-        type={presence}
-        only_on_submit
-    %}
-</div>
+<div class="username-password">
+    <div class="form-group">
+        <label for="username" class="control-label">{_ Username _}</label>
+        <input class="form-control" type="text" id="username" name="username" value="" autofocus="autofocus" autocomplete="on" />
+        {% validate id="username"
+            type={presence}
+            only_on_submit
+        %}
+    </div>
 
-<div class="form-group">
-    <label for="password" class="control-label">{_ Password _}</label>
-    <input class="form-control" type="password" id="password" name="password" value="" autocomplete="off" />
-    {% validate id="password"
-        type={presence}
-        only_on_submit
-    %}
+    <div class="form-group username-password">
```

Comment:
The class username-password is also set on the parent, is this duplication intentional?

Reply:
Yes, can be removed.